app/grants/templates/grants/landing/round_by_type.html

```diff
@@ -1,5 +1,18 @@
-{% load humanize i18n grants_extra %}
+{% load humanize i18n grants_extra static %}
 {% for active_round in active_rounds %}
+{% if active_round.display_text == "Gitcoin Building Gitcoin" %}
+
+{# Hard-coding the Gitcoin Building Gitcoin mini banner ... #}
+<div class="col-12 col-lg-6">
+  <a
+    href="https://grants.gtcdao.net/"
+    class="active-clr-banner d-block text-decoration-none position-relative mt-3"
+  >
+    <img alt="Gitcoin Building Gitcoin"  src="{% static 'landingpage/gr12_small_banner.png' %}" class="w-100">
+  </a>  
+</div>
+
+{% else %}
```

Comment:
@nutrina i don't think we'll have an active_round called `Gitcoin Building Gitcoin`
I'd say just throw it below https://github.com/gitcoinco/web/blob/master/app/grants/templates/grants/landing/active_clr_round.html#L22

Reply:
@thelostone-mc Have moved the code as you instructed. But it does not really look good in mobile. Will continue to tweak it a bit ...

Reply:
Haha alright 🙌

Reply:
@nutrina Should be final (see the screenshots). It is not perfect, but better than it was.